### PR TITLE
feat(graphql-field): Added `created_by` field in frontend

### PR DESCRIPTION
**Pull request by jatinriverlane**
# "CreatedBy" Field in Frontend
Added created_by field in frontend for Experiment List and Experiment Details page.

## ⚠️  Breaking Changes
There are no breaking changes in this PR

## ✔️ Checklist
- [x] **Code Review**: I have reviewed my changes to ensure they meet the project's coding standards and guidelines.
- [x] **Sanity Checks**:  I have performed basic sanity checks to verify the correctness of my changes.
- [x] **Testing**: Tests have been added or updated to cover the changes made in this pull request, and all tests passed successfully.
- [x] **Style Guidelines**: My code follows the style guidelines of this project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -132,4 +132,5 @@ dmypy.json
 
 
 # Other unneeded files
-.DS_Store
\ No newline at end of file
+.DS_Store
+frontend_build
```
Comment (safeamiiir) [inline .gitignore:136]: it was not already in the ignore file! 😅

**Comment by safeamiiir** [comment]
Just before you merge, we need to make it updated with the latest main changes for GitHub workflow that has fixed eslint error.

File: aqueductcore/frontend/src/components/organisms/ExperimentDetails/index.tsx [hunk 2]
```diff
@@ -385,11 +385,10 @@ function ExperimentDetails({ experimentDetails }: ExperimentDetailsProps) {
         </Grid>
         <Grid item>
           <List>
-            {/* //todo: author's name need to be provided in the backend first */}
-            {/* <ListItem sx={{ pl: 1, pr: 1 }}>
+            <ListItem sx={{ pl: 1, pr: 1 }}>
```
Comment (samiralavi) [inline aqueductcore/frontend/src/components/organisms/ExperimentDetails/index.tsx:388]: Are these variable names ok?

**Review by samiralavi — APPROVED** [review]
Looks good @jatinriverlane

File: aqueductcore/frontend/src/components/organisms/ExperimentDetails/index.tsx (hunk 2 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/ExperimentDetails/index.tsx:388]: It's a Frontend standard way of doing inline CSS with libraries like MUI or Tailwind.
they are keywords actually.

**Pull request merged**